Freshli/LibYearCalculator.cs

```diff
@@ -19,7 +19,14 @@ IManifest manifest
       var result = new LibYearResult();
 
       foreach (var package in Manifest) {
-        var latestVersion = Repository.LatestAsOf(date, package.Name);
+        VersionInfo latestVersion;
+        try {
+          latestVersion = Repository.LatestAsOf(date, package.Name);
+        } catch (Exception e) {
+          _logger.Info($"Skipping {package.Name}: {e.Message}");
+          continue;
+        }
+
```

Comment:
I'm currently handling all exceptions the same, but that may change, hence throwing two types of exceptions.